selfdrive/car/ford/carcontroller.py

```diff
@@ -89,9 +90,11 @@ def update(self, CC, CS, now_nanos):
       gas = accel
       if not CC.longActive or gas < CarControllerParams.MIN_GAS:
         gas = CarControllerParams.INACTIVE_GAS
+      else:
+        gas = 2.5
 
       stopping = CC.actuators.longControlState == LongCtrlState.stopping
-      can_sends.append(create_acc_msg(self.packer, self.CAN, CC.longActive, gas, accel, stopping))
+      can_sends.append(create_acc_msg(self.packer, self.CAN, CC.longActive, gas, accel, stopping, v_ego_kph=40 * CV.KPH_TO_MS))
```

Comment:
is converting to m/s intentional? looking at logs I'm pretty sure this signal is kph with the stock system